ci/version_var.sh

```diff
@@ -25,5 +25,5 @@ else
   PROJECT_PKG_REPO=${RELEASE_REPO}
 fi
 
-export HUB_AUTH_TAG=$PROJECT_VERSION
-export HUB_AUTH_PKG=${PROJECT_PKG_REPO}/${BASE_AUTH_PKG_NAME}
+export AUTH__TAG=$PROJECT_VERSION
```

Comment:
Is this expected to have double underscores in the name of this variable?

Reply:
[fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)